client/dashboard/sites/emails/index.tsx

```diff
@@ -1,19 +1,205 @@
-import { siteBySlugQuery } from '@automattic/api-queries';
+import { Email, SiteDomain } from '@automattic/api-core';
+import { emailsQuery, siteBySlugQuery, siteDomainsQuery } from '@automattic/api-queries';
 import { useQuery } from '@tanstack/react-query';
-import { __ } from '@wordpress/i18n';
+import { useNavigate } from '@tanstack/react-router';
+import {
+	ExternalLink,
+	Icon,
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+} from '@wordpress/components';
+import { DataViews, filterSortAndPaginate } from '@wordpress/dataviews';
+import { __, sprintf } from '@wordpress/i18n';
+import { next, wordpress } from '@wordpress/icons';
+import { useMemo, useState } from 'react';
+import { purchasesRoute } from '../../app/router/me';
 import { siteRoute } from '../../app/router/sites';
+import { DataViewsCard } from '../../components/dataviews-card';
 import { PageHeader } from '../../components/page-header';
 import PageLayout from '../../components/page-layout';
+import GoogleLogo from './resources/google-logo';
+import type { View } from '@wordpress/dataviews';
+import type React from 'react';
+
+import './styles.scss';
+
+const fields = [
+	{
+		id: 'emailAddress',
+		label: __( 'Email Address' ),
+		enableGlobalSearch: true,
+		render: ( { item }: { item: Email } ) => {
+			let iconEl = <Icon icon={ wordpress } size={ 28 } className="professional-email-icon" />;
+			if ( item.type === 'forwarding' ) {
+				iconEl = <Icon icon={ next } size={ 28 } className="email-forwarder-icon" />;
+			}
+
+			if ( item.type === 'mailbox' && item.provider === 'google_workspace' ) {
+				iconEl = <GoogleLogo size={ 24 } className="google-workspace-email-icon" />;
+			}
+
+			return (
+				<HStack spacing={ 4 } justify="flex-start">
+					<div className="email-icon-wrapper">{ iconEl }</div>
+					{ item.type === 'mailbox' ? (
+						<ExternalLink href={ `https://mail.${ item.domainName }` }>
+							{ item.emailAddress }
+						</ExternalLink>
+					) : (
+						<VStack justify="flex-start">
+							<span>{ item.emailAddress }</span>
+							<span className="text-muted">
+								{ sprintf(
+									/* translators: %s is the email messages will be forwarded to. */
+									__( 'Forwards to %s' ),
+									item.forwardingTo
+								) }
+							</span>
```

Comment:
For some reason, this text does not appear with the same muted style as in the Figma design. There is more separation between the email and the muted text.

Figma | Current branch
--- | ---
<img width="452" height="104" alt="Screenshot 2025-09-22 at 10 50 57" src="https://github.com/user-attachments/assets/61b28963-63f4-4a07-9315-4c001f823b1e" /> | <img width="329" height="60" alt="Screenshot 2025-09-22 at 10 51 02" src="https://github.com/user-attachments/assets/76b8bf50-8fa8-438b-b01f-ac042e9b32e1" />

Reply:
We will make sure fields are like the designed versions here: https://linear.app/a8c/issue/DOTDASH-474/hosting-dashboard-emails-implement-dataview-fields